app/controllers/katello/api/v2/host_tracer_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Katello
   class Api::V2::HostTracerController < Api::V2::ApiController
-    before_action :find_host
+    include Katello::Concerns::FilteredAutoCompleteSearch
+    before_action :find_host, :only => [:index, :auto_complete_search]
```

Comment:
Was it intentional to restrict find_host to only these methods? I think `resolve` needs @host to be defined.